Content.Client/Entry/EntryPoint.cs

```diff
@@ -72,6 +73,7 @@ public sealed class EntryPoint : GameClient
         [Dependency] private readonly ILogManager _logManager = default!;
         [Dependency] private readonly DebugMonitorManager _debugMonitorManager = default!;
         [Dependency] private readonly TitleWindowManager _titleWindowManager = default!;
+        [Dependency] private readonly ISharedContentMarkupTagManager _contentMarkupTagManager = default!;
```

Comment:
<_< what is that? i mean EntryPoint class... and why is manager needed here...

Reply:
Very good question; I think this is from me trying to get the content markup working, finding another fix and forgetting to remove this line.